src/.eslintrc.json

```diff
@@ -55,6 +55,27 @@
       { "ignoreRestSiblings": true }
     ],
     "@typescript-eslint/consistent-type-imports": ["warn"],
-    "react-hooks/exhaustive-deps": ["error"]
-  }
+    "react-hooks/exhaustive-deps": ["error"],
+    "sonarjs/no-duplicate-string": ["off"],
+    "sonarjs/cognitive-complexity": ["warn", 15],
+    "sonarjs/no-collapsible-if": ["warn"],
+    "sonarjs/prefer-single-boolean-return": ["warn"],
+    "sonarjs/no-identical-functions": ["warn"],
+    "sonarjs/no-small-switch": ["warn"],
+    "sonarjs/no-nested-template-literals": ["warn"]
```

Comment:
My one worry is that we are starting to get a more and more complex eslint rule, and I just dont want to end up in the scenario we had before, where the code was littered with `eslint-ignore` statements all over the place. If we introduce these rules, we need to stick by them and fix the issues. 

That said, I am all for enabling this, since we anyway already have them (just hidden in Sonarcloud) 😅 I wasnt aware eslint extensions for these existed.

Reply:
Agree on disabling the `sonarjs/no-duplicate-string`. Could be valuable in some cases I guess, but would surely be some noise we cant really fix (and then we're back to adding eslint-disable statements which I really dont want).